modules/kn-service-offline-create.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="creating-an-offline-service_{context}"]
+= Creating an offline service
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have installed `kn` CLI.
+
+.Procedure
+
+. Create an offline Knative service:
++
+[source,terminal]
+----
+$ kn service create event-display \
+    --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest \
+    --target ./ \
+    --namespace test
+----
++
+[source,terminal]
+----
+Service 'event-display' created in namespace 'test'.
+----
++
+* The `--namespace test` option places the new service in the `test` namespace.
++
+If you do not use `--namespace`, the service is created in the `default` namespace.
```

Comment:
There's a possibility of following case:
1. User is logged in into the cluster. There's active `~./kube/config` entry with a current namespace. (implied by the developers' flow working with Knative on some kind of cluster)
2. Therefore when using `--target` without `--namespace` flag, the namespace will be recognized from current session

Does it make sense to explain further?
```
If you do not use `--namespace`, and you have active login session to OpenShift cluster, the current namespace is used. Otherwise the name fallbacks to the `default` namespace.
```